pkg/wallet/wallet.go

```diff
@@ -29,12 +34,12 @@ type key struct {
 }
 
 // New return new instance of wallet implementation
-func New(storeProvider storage.Provider) (*BaseWallet, error) {
+func New(storeProvider storage.Provider, crypter crypto.Crypter) (*BaseWallet, error) {
```

Comment:
We aren’t using the context / provider mechanism here?

Reply:
I add TODO